core/controllers/profile_test.py

```diff
@@ -301,14 +301,69 @@ def test_contribution_datetime(self):
             '/profilehandler/data/%s' % self.USERNAME)
         self.assertEqual(response_dict['first_contribution_datetime'], None)
 
-        #Update the first_contribution_datetime to the current datetime.
+        # Update the first_contribution_datetime to the current datetime.
         current_datetime = datetime.datetime.utcnow()
         user_services.update_first_contribution_datetime(
             self.user_id,current_datetime)
 
-        #Test the contribution date correctly changes to set date time.
+        # Test the contribution date correctly changes to set date time.
         response_dict = self.get_json(
             '/profilehandler/data/%s' % self.USERNAME)
         self.assertEqual(
             response_dict['first_contribution_datetime'],
             utils.get_time_in_millisecs(current_datetime))
+
+
+class UserContributionsTests(test_utils.GenericTestBase):
+
+    USERNAME_A = 'a'
+    EMAIL_A = 'a@example.com'
+    USERNAME_B = 'b'
+    EMAIL_B = 'b@example.com'
+    EXP_ID_1 = 'exp_id_1'
+
+    def test_zero_count(self):
+        # Check that the profile page for a user with no contributions shows
+        # that they have 0 created/edited explorations.
+
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        response_dict = self.get_json(
+            '/profilehandler/data/%s' % self.USERNAME_A)
+        self.assertEqual(response_dict['created_explorations_count'], 0)
+        self.assertEqual(response_dict['edited_explorations_count'], 0)
+
+    def test_created_count(self):
+        # Check to see a user with 1 created explorations
```

Comment:
"Ditto for the other comments" in my previous set of comments meant that these comments should be updated as well.

For example, this one should be: "Check that the profile page for a user who has created a single exploration shows 1 created and 1 edited exploration."